### PR TITLE
Alistair/ot bump updates

### PR DESCRIPTION
**Pull request by cfrantz**
Here are some of the updates I alluded to in my review feedback for tock#3056.

I ran this against opentitan@05dcfbd00ca893dba034b468d1754f3f50780080 and I can at least get the kernel booting on my CW310 board.  The tests still don't pass (AES dies in the middle of the CBC test).

I've not tested the verilator or qemu flows.

### CODE REVIEW

File: boards/opentitan/earlgrey-cw310/Makefile [hunk 1]
```diff
@@ -40,23 +40,38 @@ qemu-app-gdb : $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
 	$(QEMU) -s -S -M opentitan -kernel $^ -bios $(OPENTITAN_BOOT_ROM) -device loader,file=$(APP),addr=0x20030000 -nographic -serial mon:stdio
 
 flash: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).bin
-	$(OPENTITAN_TREE)/util/fpga/cw310_loader.py --firmware $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).bin
+	$(call check_defined, OPENTITAN_TREE)
+	$(OPENTITAN_TREE)/bazelisk.sh run //sw/host/opentitantool -- \
```
Comment (alistair23) [inline boards/opentitan/earlgrey-cw310/Makefile:44]: I would prefer not to call bazel here. I just tested this and bazel first had to download and install something, it makes the entire process clunky

File: boards/opentitan/earlgrey-cw310/layout.ld [hunk 2]
```diff
@@ -12,8 +12,28 @@ MPU_MIN_ALIGN = 1K;
...
+        . = . + 4;   /* version_minor */
+        . = . + 4;   /* security_version */
+        . = . + 8;   /* timestamp */
+        . = . + 32;  /* binding_value */
+        . = . + 4;   /* max_key_version */
+        . = . + 4;   /* code_start */
+        . = . + 4;   /* code_end */
+        LONG(_stext - ORIGIN(rom)); /* . = . + 4; entry_point */
+        . = . + 128; /* padding */
+        /* size = 1024 bytes */
```
Comment (alistair23) [inline boards/opentitan/earlgrey-cw310/layout.ld:36]: Thanks for this.

This doesn't work for me though. I had to change `LONG(_stext - ORIGIN(rom));` to LONG(_stext);` to get it to work.

I squashed this change into the single update commit and added your SoB line. Let me know if you would rather me split out the patch

**Review by alistair23 — COMMENTED** [review]
Thanks for this!

Apparently I wasn't following my own repo, so GitHub never notified me of the PR!

File: boards/opentitan/earlgrey-cw310/layout.ld (hunk 2 above)
Comment (cfrantz) [reply boards/opentitan/earlgrey-cw310/layout.ld:36]: You need to update past opentitan@1437bb418ef3b61edba0060a25c5c5add8902840.  Prior to that commit, the test_rom did not interpret the manifest header in compliance with the [manifest](https://docs.opentitan.org/sw/device/silicon_creator/rom_ext/docs/manifest/) definition.

**Pull request closed without merging**